tuxemon/core/components/sprite.py

```diff
@@ -470,7 +470,10 @@ def _determine_cursor_movement_horizontal(self, index, event):
         :param event: pygame.Event
         :returns: New menu item offset
         """
-        if not len(self):
```

Comment:
fix bug when using the arrow keys in menus that contain just one item, or if all items are disabled.